http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -141,7 +139,7 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
               cloudContext,
               userInfo,
               req,
-              RuntimeSamResourceId(samResource.resourceId),
+              RuntimeSamResourceId(vmSamResourceId.toString),
```

Comment:
we were previously saving the workspaceId as the runtime's sam id, which was not correct